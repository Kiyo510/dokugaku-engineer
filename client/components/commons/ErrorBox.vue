<template>
  <div class="error-box">
    <slot />
  </div>
</template>

<style lang="scss" scoped>
.error-box {
  background-color: $color-red3;
  border-radius: 0.4rem;
  color: $color-red1;
  font-size: $font-size-sm;
  font-weight: 700;
  margin-bottom: 3.5rem;
  padding: 1.5rem 3rem;
}
</style>

<script>
export default {}
</script>
