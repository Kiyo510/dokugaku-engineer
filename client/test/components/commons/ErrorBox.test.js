import { mount } from '@vue/test-utils'
import ErrorBox from '~/components/commons/ErrorBox.vue'

describe('components/commons/ErrorBox.vue', () => {
  test('match snapshot', () => {
    const wrapper = mount(ErrorBox)
    expect(wrapper.element).toMatchSnapshot()
  })
})
