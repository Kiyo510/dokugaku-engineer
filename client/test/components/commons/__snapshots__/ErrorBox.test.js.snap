// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/commons/ErrorBox.vue match snapshot 1`] = `
<div
  class="error-box"
/>
`;
